packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -49,24 +55,51 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
     @state()
     private _isModalOpen = false;
 
+    @state()
+    private locale: CookieBannerLocale = defaultLocale;
+
```

Comment:
[Main change] Made property private and exposed new properties `Tenant` and `Language` instead.